src/new-components/inputs/index.tsx

```diff
@@ -0,0 +1,266 @@
+import styled from '@emotion/styled'
+import { colorsV2 } from '@hedviginsurance/brand'
+import { DownArrow } from 'components/icons/DownArrow'
+import { WarningIcon } from 'components/icons/Warning'
+import { Field, GenericFieldHTMLAttributes } from 'formik'
+import * as React from 'react'
+import InputMask from 'react-input-mask'
+
+interface Mask {
+  name: string
+  mask: string
+  sanitize: (value: string) => string
+}
+
+export const masks: { [key: string]: Mask } = {
+  zipCode: {
+    name: 'ZipCode',
+    mask: '999 99',
+    sanitize: (value) => value.replace(/\s+/, ''),
+  },
+  area: {
+    name: 'Area',
+    mask: '99999 m2',
+    sanitize: (value) => value.replace(/[\s\u200b]+m2/, ''),
```

Comment:
what is `\u200b`? if you mean `²` that's technically `\u00b2` no?

Reply:
`\u200b` the unicode character "Zero width space" 😎 https://www.fileformat.info/info/unicode/char/200b/index.htm

It was a hacky thing I did in order to get the masking to work... Otherwise, the `m2` was only added when you had filled it in if I remember correctly

Reply:
React-input-mask uses a `_` as mask char, so in order to remove it, but still keeping the `m2` I used an invisible character

Reply:
Really clever, i will just put it as a named constant so we remember this in the future